.coafile

```diff
@@ -34,6 +34,6 @@ max_lines_per_file = 1000
 [TODOS]
 enabled = False
 bears = KeywordBear
-
+language = python3
```

Comment:
it kind of is related since we want the coafile to work with the dev version of coala bears always. See also https://github.com/coala-analyzer/coala/issues/2709 though :/